nncf/openvino/graph/model_transformer.py

```diff
@@ -610,7 +640,9 @@ def _apply_bias_insertion_transformations(
             node_output_port = node.output(transformation.target_point.port_id)
             node_output_source_ports = node_output_port.get_target_inputs()
 
-            bias_const_node = opset.constant(transformation.bias_value, dtype=node.get_element_type().to_dtype())
+            bias_const_node = OVModelTransformer._create_constant(
+                transformation.bias_value, dtype=node.get_element_type().to_dtype(), name=f"{node_name}/bias"
```

Comment:
Why do we need `numpy.dtype` here?

Reply:
I also think that we should use 
```suggestion
                transformation.bias_value, dtype=node_output_port.get_element_type(), name=f"{node_name}/bias"
```
because for nodes with multiple outputs `node.get_element_type()` may throw an error

Reply:
Good catch!